sample/src/main/svg/ic_laptop.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="24pt" height="24pt" viewBox="0 0 24 24">
+<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24">
```

Comment:
Yeah I need to investigate that one. Last time I checked it was throwing an exception. But looking at the source code, there should be some kind of support: https://android.googlesource.com/platform/tools/base/+/master/sdk-common/src/main/java/com/android/ide/common/vectordrawable/VdParser.java#379
I'll try to get to the bottom of this and see if I can create a bug report for the AS project.